Sources/Orbit/Components/InputField.swift

```diff
@@ -133,7 +134,7 @@ public struct InputField<Value>: View where Value: LosslessStringConvertible {
                 }
                 .accessibility(addTraits: .isButton)
                 .accessibility(.inputFieldPasswordToggle)
-                .opacity(state == .disabled ? 0 : 1)
+                .opacity(isEnabled ? 1 : 0)
```

Comment:
This won't ever happen if `isEnabled` is `false`.

Reply:
Not sure I understand. We want to hide the eye icon toggle in case the field is disabled (the same condition as before, only reverted).